dotnet/src/Connectors/Connectors.Memory.PgVector/PostgresCollection.cs

```diff
@@ -64,8 +63,11 @@ public PostgresCollection(NpgsqlDataSource dataSource, string name, bool ownsDat
     /// <param name="options">Optional configuration options for this class.</param>
     public PostgresCollection(string connectionString, string name, PostgresCollectionOptions? options = default)
 #pragma warning disable CA2000 // Dispose objects before losing scope
-        : this(PostgresUtils.CreateDataSource(connectionString), name, ownsDataSource: true, options)
+        : this(PostgresUtils.CreateDataSource(connectionString), name, new(options)
```

Comment:
Nit: for readability, I'd widen the #pragma scope around the entire constructor (it's not like there's any other thing here that could get accidentally suppressed)